docker/jenkins.yml

```diff
@@ -62,6 +62,10 @@ jenkins:
         environments:
         - name: "TDR_IDENTITY_POOL_ID"
           value: "${/mgmt/identitypoolid_intg}"
+        - name: "INTG_ACCOUNT"
+          value: "${/mgmt/intg_account}"
```

Comment:
Might sound silly, but could these be `intg_aws_account_number` and `staging_aws_account_number`? I looked at the code before reading the PR description, and my first guess was that this was an IAM account, then possibly a Keycloak account.

Reply:
Yeah sure. I've kept them upper case though just to match the others.